### PR TITLE
added default client headers

### PR DESCRIPTION
**Pull request by logileifs**
as discussed in https://github.com/vinissimus/async-asgi-testclient/issues/19

### CODE REVIEW

File: async_asgi_testclient/testing.py [hunk 1]
```diff
@@ -158,8 +163,10 @@ def websocket_connect(self, path, extra_headers=None):
         input_queue: asyncio.Queue[dict] = asyncio.Queue()
         output_queue: asyncio.Queue[dict] = asyncio.Queue()
 
+        if headers is not None:
+            self.headers.update(headers)
```
Comment (masipcat) [inline async_asgi_testclient/testing.py]: Sorry, I think my comment https://github.com/vinissimus/async-asgi-testclient/issues/19#issuecomment-546525951 was ambiguous. My idea is to merge the headers provided in open() method with the global headers (but without modifying the global headers). 

I meant to say something like
```python
merged_headers = self.headers.copy()
merged_headers.update(headers)
...
make_test_headers_oath_query_string(..., merged_headers)
```

File: async_asgi_testclient/tests/test_testing.py [hunk 2]
```diff
@@ -175,6 +183,17 @@ class Echo(WebSocketEndpoint):
         assert resp.cookies.get_dict() == {"my-cookie": "", "my-cookie-2": ""}
         assert resp.status_code == 200
 
+        client.headers = {"Authorization": "mytoken"}
+        resp = await client.get("/echoheaders")
```
Comment (masipcat) [inline async_asgi_testclient/tests/test_testing.py]: Can you add a header in the `get()` to test that headers are merged?

File: async_asgi_testclient/testing.py (hunk 1 above)
Comment (logileifs) [reply async_asgi_testclient/testing.py]: Ahh I see, I'll change it

**Comment by masipcat** [comment]
Released v1.1.3

**Pull request merged**